rvv-intrinsic-rfc.md

```diff
@@ -348,70 +348,83 @@ vmerge.vvm vd, vs2, vs1, v0:
 vint8m1_t vmerge_vvm_i8m1_m(vbool8_t mask, vint8m1_t vs2, vint8m1_t vs1, size_t vl);
```

Comment:
As suggested in https://github.com/riscv-non-isa/rvv-intrinsic-doc/issues/126, I think we should consider moving the `mask` argument for consistency with other intrinsics. We could do this here in the new "policy" intrinsics, since they use `_ta` and `_tu`, and update the legacy API later. Just a proposal.

Reply:
I agree. I will move it.